### ISSUE TITLE
Font Size

### ISSUE THREAD

**Issue by QueenUlyana** [arsenetar/dupeguru#1132]
The font size does not change. I set different values and restarted the program, but the font does not change at all.

Windows 10 LTSC
21H2
29.‎01.‎2023
19044.2486
Windows Feature Experience Pack 120.2212.4190.0

dupeGuru [4.3.1]

debug.log
2023-06-08 07:47:42,594 - WARNING - Error while loading C:/Users/Maxim/AppData/Roaming/Hardcoded Software/dupeGuru\exclude_list.xml: [Errno 2] No such file or directory: 'C:/Users/Maxim/AppData/Roaming/Hardcoded Software/dupeGuru\\exclude_list.xml'

**Comment by arsenetar** [comment]
The font size option in the options dialog only changes the font size in the results window, font in the rest of the application stays the same size. The next scan will use the changed font size in the results window without needing to restart.

**Comment by QueenUlyana** [comment]
> The font size option in the options dialog only changes the font size in the results window, font in the rest of the application stays the same size. The next scan will use the changed font size in the results window without needing to restart.

This is bad, can you make the font change throughout the program?

**Comment by Lew-Rockwell-Fan** [comment]
I agree this is a serious issue.  I'm trying dupeguru under Debian Bookworm (stable) and theming doesn't work worth a hoot. Of course, it's not alone. Fully half of qt apps don't theme worth a hoot in gnu/linux distros. I tried kvantum and qt5ct. Neither one changes Dupeguru's ui font or colors. There is no config file I can locate. For now I'm giving up and looking for something else.
